Analysis/Tasks/PWGCF/FemtoDream/include/FemtoDream/FemtoDreamContainer.h

```diff
@@ -38,15 +39,25 @@ enum Observable { kstar };
 class FemtoDreamContainer
 {
  public:
-  void init(o2::framework::HistogramRegistry* registry, femtoDreamContainer::Observable obs)
+  template <typename T>
+  void init(o2::framework::HistogramRegistry* registry, femtoDreamContainer::Observable obs, T& kstarBins, T& multBins, T& kTBins, T& mTBins)
   {
     mHistogramRegistry = registry;
     std::string femtoObs;
     if (mFemtoObs == femtoDreamContainer::Observable::kstar) {
-      femtoObs = "#it{k}^{*} (GeV/#it{c})";
+      femtoObs = "#it{k*} (GeV/#it{c})";
     }
-    mHistogramRegistry->add("relPairDist", ("; " + femtoObs + "; Entries").c_str(), o2::framework::kTH1F, {{5000, 0, 5}});
+    std::vector<double> tmpVecMult = multBins;
+    framework::AxisSpec multAxis = {tmpVecMult, "Multiplicity"};
+    framework::AxisSpec femtoObsAxis = {kstarBins, femtoObs.c_str()};
+    framework::AxisSpec kTAxis = {kTBins, "#it{k}_{T} (GeV/#it{c})"};
+    framework::AxisSpec mTAxis = {mTBins, "#it{m}_{T} (GeV/#it{c}^{2})"};
+
+    mHistogramRegistry->add("relPairDist", ("; " + femtoObs + "; Entries").c_str(), o2::framework::kTH1F, {femtoObsAxis});
     mHistogramRegistry->add("relPairkT", "; #it{k}_{T} (GeV/#it{c}); Entries", o2::framework::kTH1F, {{5000, 0, 5}});
```

Comment:
here I would also put the kTAxis

Reply:
For the second histograms? 
Sure. I was not sure if we needed a finer binning for a basic kT histogram (that actually we never used so far), but now that is configurable from the running step there is no more need for hardcoding